pkg/volume/fc/fc_util.go

```diff
@@ -101,15 +140,28 @@ func scsiHostRescan(io ioHandler) {
 	}
 }
 
+// check scsi disk is really exist and is attached to the node using SCSI Inquiry
+func checkDiskExist(b fcDiskMounter, disk, wwid string) (bool, error) {
+	out, err := b.plugin.execCommand("/lib/udev/scsi_id", []string{"--replace-whitespace", "--page=0x83", "--whitelisted", "--device=" + disk})
```

Comment:
According to https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/5/html/Online_Storage_Reconfiguration_Guide/persistent_naming.html, 
it could also be page 0x80.